src/test/java/teammates/test/cases/ui/browsertests/AdminEmailPageUiTest.java

```diff
@@ -66,22 +79,169 @@ private void testCompose() {
+    
+    private boolean isEmailComposePageCorrect() {
+        return emailPage.isElementPresent(By.id("addressReceiverEmails"))
+            && emailPage.isElementPresent(By.name("emailsubject"))
+            && emailPage.isElementPresent(By.className("mce-tinymce"))
+            && emailPage.isElementPresent(By.id("adminEmailBox"))
+            && emailPage.isElementPresent(By.id("composeSubmitButton"))
+            && emailPage.isElementPresent(By.id("composeSaveButton"));
+    }
+    
+    private boolean hasStatusMessageNoRecipient() {
+        return emailPage.getStatus().equals("Error : No reciver address or file given");
+    }
+    
+    private boolean hasStatusMessageRecipientEmailFormatError() {
+        return emailPage.getStatus().contains("\"recipient\" is not acceptable to TEAMMATES as an email "
+                                            + "because it is not in the correct format. An email address "
+                                            + "contains some text followed by one '@' sign followed by "
+                                            + "some more text. It cannot be longer than 45 characters. "
+                                            + "It cannot be empty and it cannot have spaces.");
+    }
+    
+    private boolean hasStatusMessageNoSubject() {
+        return emailPage.getStatus().equals("\"\" is not acceptable to TEAMMATES as email subject because "
+                                          + "it is empty. The value of email subject should be no longer "
+                                          + "than 200 characters. It should not be empty.");
+    }
+    
+    private boolean hasNoErrorMessage() {
+        return !(emailPage.isElementPresent(By.className("alert-danger")));
+    }
+    
+    private boolean hasStatusMessageSaveSuccess() {
+        return emailPage.getStatus().equals("Email draft has been saved");
+    }
+    
+    /**
+     * This method only checks if the email sent data table is displayed correctly
+     * i.e, table headers are correct
+     * It does not test for the table content
+     */
+    private boolean isEmailSentDataDisplayCorrect() {
+        if (emailPage.isElementPresent(By.className("table"))) {
+            return isEmailSentTableHeaderCorrect();
+        } else {     
+            return false;
+        }
+    }
+
+    private boolean isEmailSentTableHeaderCorrect() {
+        int numColumns = emailPage.getNumberOfColumnsFromDataTable(0); // 1 table
+        
+        if (numColumns != ADMIN_EMAIL_SENT_TABLE_NUM_COLUMNS) {
+            return false;
+        }
+        
+        List<String> expectedSessionTableHeaders = Arrays.asList("Action",
+                                                                 "Address Receiver",
+                                                                 "Group Receiver",
+                                                                 "Subject",
+                                                                 "Date");
```

Comment:
I think using two lines for this (break at the `=`?) will be just as readable.
Each individual string isn't all that long.
Likewise for the other versions.